eslint.config.js

```diff
@@ -0,0 +1,78 @@
+/*
```

Comment:
The migration of the configuration files was based on https://eslint.org/docs/latest/use/configure/migration-guide#migrate-your-config-file and https://typescript-eslint.io/getting-started/.